eng/Packages.Data.props

```diff
@@ -208,7 +208,7 @@
     <PackageReference Update="Newtonsoft.Json" Version="10.0.3" />
     <PackageReference Update="NSubstitute" Version="3.1.0" />
     <PackageReference Update="NUnit" Version="3.13.2" />
-    <PackageReference Update="NUnit3TestAdapter" Version="3.17.0" />
+    <PackageReference Update="NUnit3TestAdapter" Version="4.0.0-beta.2" />
```

Comment:
Similar to @pakrym's feedback I would like to avoid taking a dependency on a prerelease if it can be helped.